spark/src/main/scala/org/apache/spark/sql/delta/DeltaErrors.scala

```diff
@@ -84,10 +87,19 @@ trait DocsPath {
       conf: SparkConf,
       relativePath: String,
       skipValidation: Boolean = false): String = {
+    if (conf == null) return "-"
```

Comment:
Do we really need this? This should be an assertion or a `require` instead. Otherwise, incorrect callers will pass without knowing why. I think it's the caller's responsibility to not pass in a `null` conf.